RELEASE.md

```diff
@@ -172,19 +178,22 @@ The Release Manager must create a release candidate branch:
     git tag -f 'x.x.x-RC-<n>' && git push origin refs/tags/x.x.x-RC-<n> && git push --force-with-lease
     ```
 
-## 3. Testing and stabilizing the release
+## 4. Testing and stabilizing the release
 
 The Release Manager must generate the release notes and announce to stakeholders the release candidate:
 
 -   [ ] 1. Generate release notes
 
-    Run the makefile target `get_release_notes` to automatically generate the release notes for the new version:
+    You will now have to generate the release notes for the new self-hosted release. To do so, you will need to first meet all the necessary requirements specified in the [release notes README file](https://github.com/codacy/release-notes-tools/blob/master/README.md#requirements). Please ensure that all the authentication tokens are properly setup before continuing.
+    Furthermore, you will also have to ensure that you have GitHub CLI installed.
+
+    After ensuring that you have everything set-up properly, you can now run the makefile target `get_release_notes` to automatically generate the release notes for the new version:
```

Comment:
We can simplify this step since the instructions you mention are described in the release notes README you link to. It's best to completely defer to that file and have only one source of truth.

```suggestion
    Make sure that you meet all the requirements specified in the [release notes README file](https://github.com/codacy/release-notes-tools/blob/master/README.md#requirements) and then run the makefile target `get_release_notes` to automatically generate the release notes for the new version:
```

Reply:
I agree with @nicklem, let's avoid duplicating the requirements here so everything is always centralized and up to date on the codacy/release-notes-tools README.

Also, just to explain why this step didn't provide much more context before, my original idea was that the output of running the Makefile target should guide the Release Manager through fixing any setup issues and point them to the requirements on the README. But I do agree that it makes more sense to leave things explicit here. :+1: